open_city_profile/settings.py

```diff
@@ -322,3 +324,6 @@
 }
 
 GDPR_AUTH_CALLBACK_URL = env("GDPR_AUTH_CALLBACK_URL")
+SOCIAL_AUTH_TUNNISTAMO_KEY = env("OIDC_CLIENT_ID")
+SOCIAL_AUTH_TUNNISTAMO_SECRET = env("OIDC_CLIENT_SECRET")
+SOCIAL_AUTH_TUNNISTAMO_OIDC_ENDPOINT = env("TOKEN_AUTH_AUTHSERVER_URL")
```

Comment:
I realised that actually the `TUNNISTAMO` part does make sense because these are used in a context that is Tunnistamo specific: fetching api tokens from the `api-tokens` endpoint (which is a Tunnistamo specific implementation). But the `SOCIAL_AUTH` part still feels weird to me.

Reply:
I'll refactor the naming. The `SOCIAL_AUTH` part was there mainly for allowing someone to easily enable Tunnistamo based login into Django admin, but that is not really the intention of this PR so point of having suitable variable names for that in this PR at least.